assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -18,4 +19,39 @@ export const tests: Tests = {
     independent: true,
     seed: 3,
   },
+  newPaymentFlow: {
```

Comment:
won't you need to add an `oldPaymentFlow` test too?

Reply:
no, one test (`newPaymentFlow`), two variants (`control`, `newPaymentFlow`)